odbcsv/tests/list-drivers-dev.txt

```diff
@@ -1,8 +1,4 @@
 ODBC Driver 17 for SQL Server
```

Comment:
No longer need to check for descriptions, just that proper driver names are setup